### PR TITLE
Add edit_2FA function

### PR DESCRIPTION
**Pull request by Kyle2142**
New convenience function to allow easy editing of 2FA settings

### CODE REVIEW

File: telethon/telegram_client.py [hunk 1]
```diff
@@ -2470,4 +2471,75 @@ def get_input_entity(self, peer):
             'Make sure you have encountered this peer before.'.format(peer)
         )
 
+    def edit_2FA(self, current_password=None, new_password=None, hint='', email=None):
```
Comment (Lonami) [inline telethon/telegram_client.py]: Python doesn't put methods on upper case at all. You don't do things like `SomeThing`, you do `some_thing`, so there's no need to do `2FA`, just `2fa`.

File: telethon/telegram_client.py [hunk 2]
```diff
@@ -2470,4 +2471,75 @@ def get_input_entity(self, peer):
...
+        email (:obj:`str`, optional):
+            Recovery and verification email.
+            Throws ``EmailUnconfirmedError`` if value differs from current one
+            Has no effect if ``new_password`` is not set
+
+        Returns:
+            ``True`` if successful, ``False`` otherwise
+        """
+        if new_password is None and current_password is None:
+            __log__.warn('Both new_password and current_password were not '\
```
Comment (Lonami) [inline telethon/telegram_client.py]: Not sure if this warning is necessary. No other method shows that on stupid cases.

File: telethon/telegram_client.py [hunk 3]
```diff
@@ -2470,4 +2471,75 @@ def get_input_entity(self, peer):
...
+                          'specified, nothing to do')
+            return False
+        
+        pass_result = self(GetPasswordRequest())
+        if isinstance(pass_result, NoPassword) and current_password:
+            __log__.warn('You supplied current_password even though this '\
+                          'account does not have one set')
+            return False  # can do "current_password = None" and continue too
```
Comment (Lonami) [inline telethon/telegram_client.py]: Yeah, that would probably be better.

File: telethon/telegram_client.py (hunk 2 above)
Comment (Lonami) [reply telethon/telegram_client.py]: Actually, maybe `raise`'ing some error if trying to clear the password without a current one would be better.

Comment (Kyle2142) [reply telethon/telegram_client.py]: manually `raise PasswordEmptyError` then?

**Comment by Lonami** [comment]
Referencing #259.

**Pull request merged**